Makefile

```diff
@@ -13,25 +8,26 @@ MANPREFIX = $(PREFIX)/share/man
 # autoreload backend: inotify/nop
 AUTORELOAD = inotify
 
+# include auto-detected libraries
+include lib_autodetect.mk
```

Comment:
This will create lib_autodetect even when running make clean. So it would be created in order to be immediately destroyed

Reply:
Do you have any standard way to avoid this then?

Reply:
Not off hand. Someone pointed it out during the original review so just echoing the concern. I'll put some thought into it later tonight